cnf-certification-test/suite_test.go

```diff
@@ -158,7 +160,14 @@ func TestTest(t *testing.T) {
 	claimhelper.UnmarshalConfigurations(configurations, claimData.Configurations)
 
 	// Run tests specs only if not in diagnostic mode, otherwise all TSs would run.
+	var env provider.TestEnvironment
 	if !diagnosticMode {
+		err = daemonset.DeployPartnerTestDaemonset()
+		if err != nil {
+			log.Errorf("Error : +%v\n", err.Error())
```

Comment:
There's a [previous comment](https://github.com/test-network-function/cnf-certification-test/commit/d66784a8d8440e9353158b9fe1f62af9de84862d#r79443611) from me about this trace:

1. The newline "\n" char is not needed in these logging functions.
2. You can just use "err". There's no need to use err.Error() here.
3. Why the "+" symbol before the %v?
4. Please use a more meaningful error message like "Error deploying partner daemonset" or similar.

Reply:
It old version , I changed. 
look- 

func DeployPartnerTestDaemonset() error {
	_, err := CreateDaemonSet(provider.DaemonSetName, provider.DaemonSetNamespace, containerName, imageWithVersion, timeout)
	if err != nil {
		logrus.Errorf("Error deploying partner daemonset %s", err.Error())
		return err
	}
	return nil
}